app/build.gradle

```diff
@@ -18,9 +24,39 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
+    // Add signing configs to sign the app with poynt keys by default, DO-6466
+    signingConfigs {
+        debug {
+//            storeFile file("keys/debug.keystore")
+//            storePassword "android"
+//            keyAlias "androiddebugkey"
+//            keyPassword "android"
+            // Using release keys as we mainly develop on devices now.
+            storeFile file("../keys/releasekey.keystore")
```

Comment:
@ahegde-godaddy  just want to make sure when you say Poynt signature - are you using the key from Copilot?  Please don't. This is a public repo. Don't forget that. By adding to your fork which is also a public repo you are already exposing this. Please delete from history ASAP.